external/js/cothority/src/calypso/calypso-rpc.ts

```diff
@@ -17,11 +17,15 @@ import { DecodeKey, OnChainSecretInstance } from "./calypso-instance";
 export class OnChainSecretRPC {
     static serviceID = "Calypso";
     private socket: IConnection;
-    private list: ServerIdentity[];
+    private readonly list: ServerIdentity[];
```

Comment:
Why is this necessary ?

Reply:
you mean it should be only `private`, but not `private readonly`? I think it was an IntelliJ suggestion, because the variable is not changed outside of the constructor. So I accepted the suggestion.